tests/runtime/call

```diff
@@ -511,3 +511,18 @@ NAME debugf_with_seq_printf
 RUN echo > /sys/kernel/debug/tracing/trace; {{BPFTRACE}} -e 'i:ms:1 { printf("printf %d", 0); debugf("debugf %d", 1); exit();}'; cat /sys/kernel/debug/tracing/trace
 EXPECT bpf_trace_printk: debugf 1
 TIMEOUT 3
+
+NAME nsecs
+PROG i:ms:1 { printf("SUCCESS %d\n", nsecs()); exit(); }
+EXPECT SUCCESS -?[0-9][0-9]*
+TIMEOUT 5
+
+NAME nsecs_boot
+PROG i:ms:1 { printf("SUCCESS %d\n", nsecs(boot)); exit(); }
+EXPECT SUCCESS -?[0-9][0-9]*
+TIMEOUT 5
+
+NAME nsecs_sw_tai
```

Comment:
We could also add a runtime test for `nsecs(tai)` which would require the TAI support by kernel (feature detection for TAI would have to be added to the test runner).

Reply:
Update to patch: [add runtime test for nsecs()](https://github.com/iovisor/bpftrace/pull/2619/commits/ba492b0887d7bc1bb3b7d8a97d0e04e8f89c945e).Thanks.